### ISSUE TITLE
Executing script 2 in VM causes Win10 guest to hang. Rebooting results in unbootable .qcow2 file

### ISSUE THREAD

**Issue by kennethnakasone** [iperov/DeepFaceLab#301]
tl;dr: Running this in a Windows VM caused the VM to hang. Upon reboot, the .qcow2 image used to boot the VM becomes unbootable. 

Hello, 
I've got (what I find to be) a weird one for you guys. I'm not sure if it's related to this application, but this problem occurs consistently, so I figure I'd mention it here as well. If someone with a similar environment could test this, I'd appreciate it (it might just be my hardware or something). 

EDIT: I just realized my GPU compute capability is too low to utilize the CUDA version of DeepFaceLab, but that should be irrelevant for this script, so I'll leave this up.

## Expected behavior
Running the script "2) extract images from video data_src" opens Commandline window, processes data_src into images and places them into workspace/data_src folder. Window then closes.

## Actual behavior
The script appears to finish processing the video, but then the VM hangs. The cursor still responds, but Windows does not respond to mouse button or keyboard input. System maintains this state for what appears to be indefinitely (at least 12hrs, anyway). Powering down VM and turning on again results in unbootable .qcow2 file. File still appears as a QEMU harddisk in the boot manager of the VM, but when selecting it for boot, the screen flashes, then returns to the boot options page. 

## Steps to reproduce
Start virtual machine using Virtual Machine Manager connected to QEMU/KVM, using GPU passthrough, while hiding hypervisor presence (to bypass NVIDIA code 34). Run script 2. 

I was able to reproduce the issue 3 times consecutively.

## Other relevant information
DeepFaceLab Build: Prebuilt DeepFaceLabCUDA10.1AVX_build_06_20_2019.exe (uploaded 06/20/2019 at 00:09 to MEGA)

System Information:
Host 
OS: Ubuntu 18.04
CPU: Xeon E3-1230v3 
Motherboard: Asus Z87-A
RAM: 2x8GB DDR3 G.Skill Ares 1600Mhz + 2x8GB DDR3 Patriot Viper 1600MHz
Storage: Crucial M500 + HGST Ultrastar 7K4000

Guest:
OS: Windows 10 Home 1809
CPU: host-model (3C6T)
Motherboard: Q35 w/ UEFI
RAM: 16GB
Storage: .qcow2 file 

I've also made sure that he .qcow2 file was large enough to hold all the images, and that the hard drive that the .qcow2 file was located on had enough physical space to accommodate it.
